behave/runner.py

```diff
@@ -495,6 +498,54 @@ def path_getrootdir(path):
     return os.path.sep
 
 
+def forEachStatusInFeatures(feature, f, param):
```

Comment:
These methods should be private in the class, so don't want to export them in the module.